genesis/vis/visualizer.py

```diff
@@ -244,3 +272,15 @@ def connected_to_display(self):
     @property
     def cameras(self):
         return self._cameras
+
+    @property
+    def camera_pos(self):
```

Comment:
Where are used these getters? Are they relevant? Not all of them are properly enforcing dtype and device.